sdk/python/feast/feature_store.py

```diff
@@ -1144,6 +1144,27 @@ def tqdm_builder(length):
                 feature_view, self.project, start_date, end_date,
             )
 
+    @log_exceptions_and_usage
+    def push(self, push_source_name: str, df: pd.DataFrame):
+        data_sources = self.registry.list_data_sources(self.project, True)
+        push_source = {
```

Comment:
shouldn't we expect there to be only one PushSource since DS are unique by name?